src/components/PipelineRun/PipelineRunDetailsView/tabs/PipelineRunDetailsTab.tsx

```diff
@@ -48,11 +48,18 @@ const PipelineRunDetailsTab: React.FC = () => {
   const { pipelineRunName, workspaceName: workspace } = useParams<RouterParams>();
   const { namespace } = useWorkspaceInfo();
   const generateSbomUrl = useSbomUrl();
-  const [pipelineRun, loaded, error] = usePipelineRun(namespace, workspace, pipelineRunName);
+
+  const recordPath = useLocation().state?.recordpath as string;
+  const [pipelineRun, loaded, error] = usePipelineRun(
+    namespace,
+    workspace,
+    pipelineRunName,
+    recordPath,
+  );
   const [taskRuns, taskRunsLoaded, taskRunError] = useTaskRuns(namespace, pipelineRunName);
 
   const snapshotStatusAnnotation =
```

Comment:
You can use useMemo() with loaded & pipelineRun

Reply:
I have updated it with useMemo()